Resources/Locale/en-US/research/components/robotics-console.ftl

```diff
@@ -0,0 +1,18 @@
+robotics-console-window-title = Robotics Console
+robotics-console-no-cyborgs = No Cyborgs!
+
+robotics-console-select-cyborg = Select a cyborg above.
+robotics-console-model = Model: {$name}
+robotics-console-designation = Designation: {$name}
+robotics-console-battery = Battery charge: {$charge}%
+robotics-console-modules = Modules installed: {$count}
+robotics-console-brain = Brain installed: {$brain ->
+    [true] Yes
+    *[false] No
+}
```

Comment:
use color and perhaps bold to make these more interesting. plain white text in a box is visually very boring.

Reply:
it a bit better now tgui real